### ISSUE TITLE
Highstate output broken when using jobs.lookup_jid

### ISSUE THREAD

**Issue by jfrank-quatral** [saltstack/salt#27723]
Using the latest Salt release, 2015.8, it seems that the highstate output is broken in some cases on the salt master.

Command: `sudo salt-run jobs.lookup_jid 20151006191510666868 --out=highstate`
Error message: http://pastebin.com/skgBc83L

The output is working when not using `--out=highstate` argument, but return as data structure and not the traditional highstate output.

**Comment by Ch3LL** [comment]
@jfrank-quatral I actually received a different error when attempting to run this command outputted below:

```python-traceback
Traceback (most recent call last):
  File "/bin/salt-run", line 10, in <module>
    salt_run()
  File "/usr/lib/python2.7/site-packages/salt/scripts.py", line 354, in salt_run
    client.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/run.py", line 40, in run
    runner.run()
  File "/usr/lib/python2.7/site-packages/salt/runner.py", line 165, in run
    False)  # Don't daemonize
  File "/usr/lib/python2.7/site-packages/salt/client/mixins.py", line 404, in _proc_function
    return self.low(fun, low)
  File "/usr/lib/python2.7/site-packages/salt/client/mixins.py", line 350, in low
    namespaced_event.fire_event(data, 'ret')
  File "/usr/lib/python2.7/site-packages/salt/utils/event.py", line 700, in fire_event
    self.print_func(tag, data)
  File "/usr/lib/python2.7/site-packages/salt/client/mixins.py", line 484, in print_async_event
    salt.output.display_output(event_data, outputter, self.opts)
  File "/usr/lib/python2.7/site-packages/salt/output/__init__.py", line 86, in display_output
    display_data = try_printout(data, out, opts)
  File "/usr/lib/python2.7/site-packages/salt/output/__init__.py", line 46, in try_printout
    return get_printout('raw', opts)(data).rstrip()
AttributeError: 'NoneType' object has no attribute 'rstrip'
```

**Comment by basepi** [comment]
Fixed this today: https://github.com/saltstack/salt/pull/27838

@Ch3LL we may have to investigate your error if you're still getting it after the fix in #27838.
